docs/src/user-guide/reference-json-search-syntax.md

```diff
@@ -191,6 +192,19 @@ characters using a `\` (backslash):
 * `{`
 * `}`
 
+### Supported escape sequences
```

Comment:
I assume all of these sequences don't require an extra backslash to escape the backslash in the sequence, right? If so, I think it's worth adding a note (`:::{note}`) in the section above that escape sequences don't require extra escapes for the backslash.